install/install.sh

```diff
@@ -55,11 +58,13 @@ echo "-------------------------------"
 echo ""
 echo "In order to configure the platform, we need some information from you. Please reply to the following questions:"
 echo ""
-read -e -r -p -s "Password of the 'root' user of MySQL: " -i "" db_root_password
+read -s -e -r -p  "Password of the 'root' user of MySQL: " -i "" db_root_password
+ echo ""
 # Verify password
+
 supress_warning=$(mysql_config_editor set --login-path=root_login --host=localhost --user=root --password "${db_root_password}") >> "$install_log" 2>&1
 while ! mysql  --login-path=root_login  -e ";" ; do
-    read -e -r -p -s "Invalid password, please retry: " -i "" db_root_password
+    read -s -e -r -p "Invalid password, please retry: " -i "" db_root_password
```

Comment:
So swapping around resolves the issue? That's curious :)

Reply:
@eshandhawan51 What are you trying to solve here? I don't think the ordering in anyway affects the function.

Reply:
It did on his distro, so it's a valid fix as far as I'm concerned.

Reply:
earlier it showed the warning  "': not a valid identifier" in the terminal so I earlier removed the -s from the read function but and the error stopped popping up it also blocked the input through the terminal 
nut later on when I applied it it worked so I didn't remove it

Reply:
technically it shouldn't but somehow it did :)